staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -538,7 +533,7 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	i18n.LoadTranslations("kubectl", nil)
 
 	// From this point and forward we get warnings on flags that contain "_" separators
-	cmds.SetGlobalNormalizationFunc(cliflag.WarnWordSepNormalizeFunc)
+	cmds.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
```

Comment:
This is refactor in https://github.com/kubernetes/kubernetes/pull/100171

Reply:
In #100171, only kube-apiserver, kube-scheduler, kube-controller-manager and cloud-provider were changed, kubectl was not changed.